src/components/molecules/Settings/Account/AccountSection/index.tsx

```diff
@@ -22,19 +24,33 @@ const items = [
 
 const ProfileSection: React.FC<Props> = ({
   email,
+  appTheme,
   hasPassword,
   updatePassword,
   updateLanguage,
+  updateTheme,
 }) => {
   const intl = useIntl();
   const [isOpen, setIsOpen] = useState(false);
   const [currentLang, setCurrentLang] = useState<string>();
-
+  const [currentThemeValue, setCurrentThemeValue] = useState<string>();
+  const [currentThemeLabel, setCurrentThemeLabel] = useState<string>();
+  const themeItems = [
+    { key: "DARK", label: intl.formatMessage({ defaultMessage: "Dark theme" }), icon: "moon" },
+    { key: "LIGHT", label: intl.formatMessage({ defaultMessage: "Light theme" }), icon: "sun" },
+  ];
   useEffect(() => {
```

Comment:
I prefer to give a more strict type 

```
type Theme = "DARK" | "LIGHT";
const themeItems: {key: Theme, label: string, icon: Icon} = [
    { key: "DARK", label: intl.formatMessage({ defaultMessage: "Dark theme" }), icon: "moon" },
    { key: "LIGHT", label: intl.formatMessage({ defaultMessage: "Light theme" }), icon: "sun" },
] as const;

Reply:
In the above code,`as const` is not required.